legion/engine/physics/colliders/physicscollider.hpp

```diff
@@ -24,7 +17,35 @@ namespace legion::physics
     {
     public:
 
-        PhysicsCollider() = default;
+        std::vector<std::unique_ptr<ConverganceIdentifier>> converganceIdentifiers;
+
+        PhysicsCollider()
+        {
+            static int colliderID = 0;
+
+            id = colliderID;
+
+            colliderID++;
```

Comment:
code above could be written as id=colliderID++; or id = colliderID; ++colliderID, little bit more optimized. but that's just me nitpicking, so I just wanted to point it out.